spark/src/main/scala/io/spark/ddf/analytics/BinningHandler.scala

```diff
@@ -1,18 +1,45 @@
 package io.spark.ddf.analytics
 
 import io.ddf.DDF
+import org.apache.spark.rdd.{DoubleRDDFunctions, RDD}
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.catalyst.expressions._
 import scala.collection.JavaConversions._
 import scala.collection.JavaConverters._
-import io.ddf.analytics.ABinningHandler
+import io.ddf.analytics.{AStatisticsSupporter, ABinningHandler, IHandleBinning}
 import io.ddf.analytics.ABinningHandler._
-import io.ddf.analytics.IHandleBinning
 import io.ddf.exception.DDFException
 import java.text.DecimalFormat
 import scala.annotation.tailrec
 import scala.Array.canBuildFrom
+import scala.collection.mutable.ArrayBuffer
+import scala.collection.JavaConverters._
 
 class BinningHandler(mDDF: DDF) extends ABinningHandler(mDDF) with IHandleBinning {
 
+
+  def parseDouble(r: Row) = try {r.get(0).toString.toDouble } catch { case _ => None }
+
+  override def getVectorHistogramImpl(columnName: String, numBins: Int): java.util.List[AStatisticsSupporter.HistogramBin] = {
+    val projectedDDF: DDF = mDDF.VIEWS.project(columnName)
+    val rdd: RDD[Row] = projectedDDF.getRepresentationHandler.get(classOf[RDD[_]], classOf[Row]).asInstanceOf[RDD[Row]]
+    val rdd1 = rdd.map(r => {try {r.get(0).toString.toDouble } catch { case _ => None }})
```

Comment:
I talked with Huan. He said try-catch significantly decreases the performance. I will check and update the code accordingly when I have cycle. At the moment, I believe it is not a stopper issue though.